runtime/capsules/runtime/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+# Licensed under the Apache-2.0 license
```

Comment:
I noticed the directory is `runtime/capsules/runtime`. That seems a bit redundant :)

Maybe `runtime/capsules` or, we think we will have different kinds of capsules in the future, `runtime/capsules/devices` or `drivers` (or something).

Reply:
Right! changed to `runtime/capsules`